content/en/synthetics/guide/identify_synthetics_bots.md

```diff
@@ -99,6 +97,14 @@ if (window._DATADOG_SYNTHETICS_BROWSER === undefined) {
 }
 ```
 
+If you use the browser variable to identify Synthetic bots on Firefox and Internet Explorer 11, Datadog cannot guarantee that the browser variable is set before your code executes. To avoid this issue, use Edge or Google Chrome.
```

Comment:
I think we should remove the recommendation to use Edge/Chrome, and instead let customers know they can use a combination of the above methods (maybe at the bottom of the page? Let me know what you think).

Reply:
```suggestion
If you use the browser variable to identify Synthetic bots on Firefox and Internet Explorer 11, Datadog cannot guarantee that the browser variable is set before your code executes. 
```